tests/post/steps/test_service_configuration.py

```diff
@@ -180,6 +186,47 @@ def restore_csc(
     )
 
 
+@then(parsers.parse(
+    "we have an alert rule '{rule_name}' in group '{group_name}' with "
+    "'{path}' equal to '{value}'"
+))
+def check_prometheus_alert_rule(
+        control_plane_ip, rule_name, group_name, path, value):
+    """Retrieve an alert rule from the Prometheus API, then
+    checks if the given path matches the value.
+    """
+
+    def _wait_prometheus_config_reload():
+        response = requests.get(
+            'https://{}:8443/api/prometheus/api/v1/rules'.format(
```

Comment:
If you're considering alert rules only (from the parser text), shouldn't you filter this query? `/api/v1/rules?type=alert`

Reply:
I tried, but IIRC it does not work.